<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Functional Testing" duration-ms="105884" started-at="2018-07-27T05:29:30Z" finished-at="2018-07-27T05:31:16Z">
    <groups>
      <group name="cucumber">
        <method signature="TestRunner.feature(cucumber.api.testng.CucumberFeatureWrapper)[pri:0, instance:com.trimble.taf.suite.tests.base.TestRunner@12468a38]" name="feature" class="com.trimble.taf.suite.tests.base.TestRunner"/>
      </group> <!-- cucumber -->
    </groups>
    <test name="Login Test Birst" duration-ms="105884" started-at="2018-07-27T05:29:30Z" finished-at="2018-07-27T05:31:16Z">
      <class name="com.trimble.taf.suite.tests.base.TestRunner">
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:com.trimble.taf.suite.tests.base.TestRunner@12468a38]" name="setUpClass" is-config="true" duration-ms="344" started-at="2018-07-27T05:29:30Z" finished-at="2018-07-27T05:29:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="FAIL" signature="feature(cucumber.api.testng.CucumberFeatureWrapper)[pri:0, instance:com.trimble.taf.suite.tests.base.TestRunner@12468a38]" name="feature" duration-ms="104997" started-at="2018-07-27T05:29:30Z" description="Runs Cucumber Feature" data-provider="features" finished-at="2018-07-27T05:31:15Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Common Tests]]>
              </value>
            </param>
          </params>
          <exception class="cucumber.runtime.CucumberException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[cucumber.runtime.CucumberException: java.lang.NullPointerException
at cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:69)
at com.trimble.taf.suite.tests.base.TestRunner.feature(TestRunner.java:43)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NullPointerException
at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:882)
at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:104)
at org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:71)
at org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:45)
at com.trimble.taf.pagefactory.global.AbstractPage.waitForElementPresent(AbstractPage.java:591)
at com.trimble.taf.pagefactory.application.MyspacesPage.verifyMyspacelbl(MyspacesPage.java:104)
at com.trimble.birst.tests.ComonTestssteps.verify_Homepage(ComonTestssteps.java:84)
at âœ½.Then I validate home page displayed correctly(commontests.feature:9)
]]>
            </full-stacktrace>
          </exception> <!-- cucumber.runtime.CucumberException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- feature -->
        <test-method status="PASS" signature="tearDownClass()[pri:0, instance:com.trimble.taf.suite.tests.base.TestRunner@12468a38]" name="tearDownClass" is-config="true" duration-ms="389" started-at="2018-07-27T05:31:15Z" finished-at="2018-07-27T05:31:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- com.trimble.taf.suite.tests.base.TestRunner -->
    </test> <!-- Login Test Birst -->
  </suite> <!-- Functional Testing -->
</testng-results>
